### ISSUE TITLE
Adjust source and content type for inventories

### ISSUE THREAD

**Issue by lpugin** [rism-digital/rism-online-issues#323]
Inventories in Muscat bring two new record types for sources to be adjusted here
https://github.com/rism-digital/muscatplus_indexer/blob/7d29c45540012cb54fd76ec3df8ed4b78ec92bb0/indexer/helpers/identifiers.py#L25
* 12: inventory
* 13: inventory_edition

12 needs to be mapped to `manuscript`, and 13 to `print` source type.

The content type shows as `Inventory`, which is fine. However, it is missing a the translation and a content type (yellow) icon.

**Comment by lpugin** [comment]
(`unspecified` source type and the blank green icon will disappear)

![image](https://github.com/user-attachments/assets/ab985ca2-cba5-42d9-b09b-5b5aa07c43b5)

**Comment by ahankinson** [comment]
This will probably disappear when the source is unpublished.

**Comment by lpugin** [comment]
Yes. And no new source type will be needed when we publish them. Only the content type.

**Comment by ahankinson** [comment]
Done.
